test-cypress.sh

```
@@ -27,18 +27,14 @@ do
 # Run tests.
-npm_script_args=""
-
 if [ -n "${gui-}" ]; then
-  npm_script_args="test-cypress"
-fi
-
-if [ -n "${spec-}" ]; then
-  npm_script_args="$npm_script_args --spec '$spec'"
+  npm run test-cypress
+elif [ -n "${spec-}" ]; then
+  npm run test-cypress-headless -- --spec $spec
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)